### DISCUSSION TITLE
DL相亲角

### DISCUSSION THREAD

**Discussion by xiepaopaoer** [Dapp-Learning-DAO/Dapp-Learning#817] [LOVE]
人间有真情，DL有真爱。

**Comment by ryang-max** [comment]
前排前排

**Comment by qijinzing** [comment]
前排占位，重在参与～ 
性别女，爱好男（最近磕cp磕的我有点怀疑了）

**Comment by kasoqian** [comment]
性别男，爱好男，找个可爱的男生，给你看我新买的夜光手表。

**Reply by ghost** [reply]
好啊，我给你唱小星星和两只老虎

**Comment by xiepaopaoer** [comment]
性别女，爱好男，身高168，98年，找个男孩子一起爬web3。

**Reply by qijinzing** [reply]
> 

98年的还需要吗？明明是我们89年才干的事情

**Reply by xiepaopaoer** [reply]
98、89是一家

**Reply by ghost** [reply]
别吵了别吵了，我不介意年龄，记得给我发邮件哦

**Reply by xiepaopaoer** [reply]
我忘记留邮箱了，kelly_xzw@163.com，欢迎来聊天

**Reply by qijinzing** [reply]
> 98、89是一家

好的，那我就留这里了 jinqizing@gmail.com

**Comment by daocore** [comment]
占个楼，蹲个媳妇儿

**Comment by ghost** [comment]
听说这里可以相亲？走过路过的单身女青年们，记得给我发邮件哦(echarts.eth@mmail.ink)，我带你在web3的海洋里像杰克和肉丝一样张开臂膀拥抱大海。

**Reply by kevervip** [reply]
我在DL等着你

**Comment by yanyanho** [comment]
各位，资料不全，择偶要求不全就算了，起码也要留全联系方式吧

**Reply by xiepaopaoer** [reply]
操心的老母亲

**Comment by kosmosr** [comment]
97年 爱好女 蹲一个💕 昵称跟群里的一样

**Reply by keienWang** [reply]
96年，爱好 美女，蹲一个爱穿黑丝的

**Reply by kosmosr** [reply]
> 96年，爱好 美女，蹲一个爱穿黑丝的
兄dei 发错位置了😂
